frontend/src/components/dev/logger.ts

```diff
@@ -0,0 +1,8 @@
+const logger = (description: string) => (arg: any) => {
+  if (process.env.NODE_ENV !== 'production') {
+    console.info(`${description}\n${arg}`);
```

Comment:
Is arg necessarilly serializable to string? Could log the arg outside the string.

Reply:
Fixed!